src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/vedtak/EfVedtakRepository.kt

```diff
@@ -7,35 +7,90 @@ import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
 import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
 import org.springframework.stereotype.Repository
 import java.sql.ResultSet
+import java.time.YearMonth
 
 @Repository
 class EfVedtakRepository(val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) {
 
     fun lagreEfVedtakshendelse(vedtakshendelse: EnsligForsørgerVedtakhendelse) {
-        val sql = "INSERT INTO efvedtakhendelse VALUES(:behandlingId, :personIdent, :stønadType)"
+        val sql = "INSERT INTO efvedtakhendelse VALUES(:behandlingId, :personIdent, :stønadType, :aar_maaned_prosessert, :versjon)" +
+            " ON CONFLICT DO NOTHING"
         val params = MapSqlParameterSource(
             mapOf(
                 "behandlingId" to vedtakshendelse.behandlingId,
                 "personIdent" to vedtakshendelse.personIdent,
-                "stønadType" to vedtakshendelse.stønadType.toString()
+                "stønadType" to vedtakshendelse.stønadType.toString(),
+                "aar_maaned_prosessert" to YearMonth.now().toString(),
+                "versjon" to 1
             )
         )
         namedParameterJdbcTemplate.update(sql, params)
     }
 
-    fun hentEfVedtakHendelse(personIdent: String): EnsligForsørgerVedtakhendelse? {
+    fun hentEfVedtakHendelse(personIdent: String): VedtakhendelseInntektberegning? {
```

Comment:
Denne er kun brukt i tester. Fin å ha for å sjekke att mappingen virker, skulle kunne fjernes og teste `hentAllePersonerMedVedtak()` nå?